kanopi/wp-vanilla-theme
# Sm gulp sass single point of entry

## Description
> As a developer, I need Gulp to start its Sass compilation with a single file, so that the load order of imports can be explicitly set to account for things like variables, functions, and mixins.

Currently Sass compiles in whatever order it finds the files as per globbing rules.

## Affected URL
n/a

## Related Tickets
n/a


## Steps to Validate
1. Before pulling this branch for testing, add a test file to`sass/components` and create a rule using one of the color variables in `sass/utilities/variables/_colors.scss`. Run `gulp sass`. It should fail with `Error: Undefined variable`.
2. Pull the branch and run `gulp sass`. It should now successfully compile without that error.
3. Run `gulp sass watch` and edit the test file to verify that the watch task still functions when files other than `style.scss` are changed.
